components/expenses/Expense.tsx

```diff
@@ -118,6 +118,7 @@ function Expense(props) {
     showFilesViewerModal: false,
   });
   const [openUrl, setOpenUrl] = useState(null);
+  const [replyingToComment, setReplyingToComment] = useState({});
```

Comment:
Maybe use null rather than an empty object, that way it will be simpler to check whether we're replying or not.

```suggestion
  const [replyingToComment, setReplyingToComment] = useState(null);
```